seccomp/README.md

```diff
@@ -1,61 +1,225 @@
+
+   ```
+   $ cd dockercon-workshop/seccomp
+   ```
+
+The remaining steps in this lab will assume that you are running commands from this `dockercon-workshop/seccomp` directory. This will be important when referencing the seccomp profiles on the various `docker run` commands throughout the lab.
+
+# <a name="test"></a>Step 2: Test a seccomp profile
+
+> **NOTE TO LAB OWNER:  THIS STEP DOES NOT CURRENTLY WORK. I'M GUESSING THIS IS BECAUSE THE SECCOMP PROFILE IS APPLIED BEFORE THE CONTAINER IS STARTED AS PER GITHUB ISSUES CITED LATER IN THE LAB GUIDE. I'M LEAVING IT IN FOR THE TIME BEING (NIGELPOULTON@HOTMAIL.COM) BUT AM RAISING THE QUESTION OF WHETHER IT COULD BE REMOVED IN FAVOUR OF STEP 2 WHICH DEMOS SIMILAR CAPABILITIES**
+
+In this step you will use the `deny.json` seccomp profile included the lab guides repo. This profile has an empty syscall whitelist meaning all syscalls will be blocked. As part of the demo you will add all *capabilities* and effectively disable *apparmor* so that you know that only your seccomp profile is preventing the syscalls.
+
+1. Use the `docker run` command to start a new container with all capabilities added, apparmor unconfined, and the `seccomp-profiles/deny.json` seccomp profile applied. Then try and run the `whoami` command form within the container.
+
+   ```
+   $ sudo docker run --rm -it --cap-add ALL --security-opt apparmor=unconfined --security-opt seccomp=seccomp-profiles/deny.json alpine sh
+
+   / # whoami
+   whoami: /: Operation not permitted
+   ```
+
+2. Exit the container.
+
+3. Inspect the contents of the `seccomp-profiles/deny/json` profile.
+
+   ```
+   $ cat seccomp-profiles/deny.json
+   {
+        "defaultAction": "SCMP_ACT_ERRNO",
+        "architectures": [
+                "SCMP_ARCH_X86_64",
+                "SCMP_ARCH_X86",
+                "SCMP_ARCH_X32"
+        ],
+        "syscalls": [
+        ]
+   }
+   ```
+
+   Notice that there are no **syscalls** in the whitelist. This means that no syscalls will be allowed from containers started with this profile.
+
+In this step you removed *capabilities* and *apparomor* from interfering, and started a new container with a seccomp profile that had no syscalls in its whitelist. You saw how this prevented all syscalls from within the container.
+
+# <a name="no-default"></a>Step 3: Run a container with no seccomp profile
+
+Unless you specify a different profile, Docker will apply the default seccomp profile to all new containers. In this step you will see how to force a new container to run without a seccomp profile.
+
+1. Start a new container with the `--security-opt seccomp=unconfined` flag so that no seccomp profile is applied to it.
+
+   ```
+   $ sudo docker run --rm -it --security-opt seccomp=unconfined alpine sh
+   ```
 
-### Writing a seccomp profile
+2. From the terminal of the container run the `whoami` command to confirm that the container works and can make syscalls back to the Docker Host.
```

Comment:
we should get them to run something like `mount -o bind /root /tmp` (which is disabled with seccomp by default) to prove that seccomp is disabled